graylog2-web-interface/src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Jumbotron, Row } from 'react-bootstrap';
+import { DocumentTitle } from 'components/common';
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import style from '!style/useable!css!./NotFoundPage.css';
+import errorPageStyles from './ErrorPage.css';
+
+class ErrorPage extends React.Component {
+  static propTypes = {
+    error: PropTypes.shape({
+      message: PropTypes.string.isRequired,
+    }).isRequired,
+    info: PropTypes.shape({
+      componentStack: PropTypes.string.isRequired,
+    }).isRequired,
+  };
+
+  componentDidMount() {
+    style.use();
+  }
+
+  componentWillUnmount() {
+    style.unuse();
+  }
+
+  render() {
+    const { error, info } = this.props;
+
+    return (
+      <DocumentTitle title="Not Found">
```

Comment:
I would write "Something went wrong" here as well. Since something went wrong and not something was not found.

Reply:
True!